### PR TITLE
added SubunitOfLandRegister replacement to properties and TOC

### PR DESCRIPTION
**Pull request by voisardf**
merge PR https://github.com/openoereb/pyramid_oereb_mfp/pull/31 first.

fixes 
- [x] issue https://github.com/openoereb/pyramid_oereb_mfp/issues/27 - subunit_of_land_register not printed on TOC-page in PDF extract

### CODE REVIEW

**Comment by voisardf** [comment]
example pdf : 
[20191011015222_extract.pdf](https://github.com/openoereb/pyramid_oereb_mfp/files/3717405/20191011015222_extract.pdf)

**Review by vvmruder — APPROVED** [review]
Thanks a lot. I hope this is sustainable and the federal specification will not rely on the attachments... -.-

**Review by kdeininger — APPROVED** [review]

File: print-apps/oereb/Report_de.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 TableofContentLabel = Übersicht ÖREB-Themen
-Topicsaffectingtherealestate_ofthemunicipalityof_ = Eigentumsbeschränkungen, welche das Grundstück %s in %s betreffen
+Topicsaffectingtherealestate_ofthemunicipalityof_ = Eigentumsbeschränkungen, welche das Grundstück %s in %s %s betreffen
```
Comment (marionb) [inline print-apps/oereb/Report_de.properties]:
```suggestion
Topicsaffectingtherealestate_ofthemunicipalityof_ = Eigentumsbeschränkungen, welche das Grundstück %s in %s%s betreffen
```

File: print-apps/oereb/pdfextract.jrxml [hunk 2]
```diff
@@ -484,11 +494,6 @@
 	</group>
 	<detail>
 		<part uuid="dbf7b612-de21-4609-8537-6538c611480c">
-			<!--
```
Comment (marionb) [inline print-apps/oereb/pdfextract.jrxml]: Can you re-add this note. It should not be removed eventhough it is just a comment.

**Review by marionb — CHANGES_REQUESTED** [review]

**Comment by voisardf** [comment]
Addressed comments. Should be ok now

**Review by marionb — APPROVED** [review]
Looks good now. Thanks!

**Comment by voisardf** [comment]
thanks! merging

**Pull request merged**